src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -493,18 +493,18 @@ public static class Diagnostics {
 			category: "Correctness",
 			defaultSeverity: DiagnosticSeverity.Error,
 			isEnabledByDefault: true
-		);
-
-		public static readonly DiagnosticDescriptor UnnecessaryAllowedListEntry = new DiagnosticDescriptor(
-			id: "D2L0057",
-			title: "Unnecessarily listed in an analyzer allowed list",
-			messageFormat: "The entry for '{0}' in '{1}' is unnecessary.",
-			category: "Cleanliness",
-			defaultSeverity: DiagnosticSeverity.Error,
-			isEnabledByDefault: true,
-			description: "Unnecessarily listed in an analyzer allowed list."
-		 );
-
+		);
+
+		public static readonly DiagnosticDescriptor UnnecessaryAllowedListEntry = new DiagnosticDescriptor(
```

Comment:
Not actually changed.  Ignore whitespace changes

Reply:
Mixed line endings in this file

![image](https://user-images.githubusercontent.com/12101120/89065835-65573780-d33a-11ea-83c5-e30697e3b990.png)